test/simple/test-module-version.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+var assert = require('assert');
+
+// Check for existence
+assert(process.config.variables.hasOwnProperty('node_module_version'));
+
+// Ensure that `node_module_version` is an Integer
+assert(!Number.isNaN(parseInt(process.config.variables.node_module_version)));
+assert.equal(process.config.variables.node_module_version, 11);
```

Comment:
Apologies, I said `assert.equal()` because I had a fuzzy notion that strictEqual() doesn't exist in v0.10.  I was probably thinking of deepStrictEqual().